src/_actions/WorkspaceActions.js

```diff
@@ -55,3 +55,11 @@ export const delFromWatchlist = symbol => {
         dispatch(workspaceUnfavorAsset(symbol));
     };
 };
+
+export const selectAssetSymbolForTrade = (newSymbol, oldSymbol) => {
+    return dispatch => {
+        LiveData.api.unsubscribeFromTick(oldSymbol);
+        LiveData.api.subscribeToTick(newSymbol);
```

Comment:
I was just thinking about your reducers as pure functions, and you are digging us deeper ;)
Just keep in mind for next refactoring!